blob/drivertest/drivertest.go

```diff
@@ -161,6 +161,9 @@ func RunConformanceTests(t *testing.T, newHarness HarnessMaker, asTests []AsTest
 	t.Run("TestList", func(t *testing.T) {
 		testList(t, newHarness)
 	})
+	t.Run("TestListWeirdKeys", func(t *testing.T) {
```

Comment:
Note to reviewer: `testKeys` used to test reading + writing + List of weird keys. I changed it to test everything except List, and added this test for List explicitly.